src/pages/operators/{contentfulOperatorAnalysis.operator__name}.tsx

```diff
@@ -398,6 +398,11 @@ const globalOverrideStyles =
       a {
         color: ${accentColor};
       }
+      
+      .heading-block {
+        background: linear-gradient(90deg, ${transparentize(0.9, accentColor)}, ${transparentize(1, theme.palette.white)}), ${transparentize(0.67, theme.palette.midtoneBrighter)};
```

Comment:
1. Isn't this missing a final closing paren to match the opening of `linear-gradient(`?
2. Let's use literal `transparent` instead of transparentizing white by 100%

Reply:
1. no, because it's linear-gradient(x, y), z
2. sure